releases/8.3/languages/en.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+return [
+    'common_header' => 'PHP 8.3 is a major update of the PHP language. Readonly amendments, Typed class constants, #[Override] attribute, Randomizer additions, performance improvements and more',
+    'documentation' => 'Doc',
+    'main_title' => 'Released!',
+    'main_subtitle' => 'PHP 8.3 is a major update of the PHP language.<br class="display-none-md">It contains many new features, including Readonly amendments, Typed class constants, Randomizer additions, performance improvements and more.',
+    'upgrade_now' => 'Upgrade to PHP 8.3 now!',
+
+    'readonly_title' => 'Readonly amendments',
+    'json_validate_title' => 'New <code>json_validate()</code> function',
+    'typed_class_constants_title' => 'Typed class constants',
+    'override_title' => '<code>#[\Override]</code> attribute',
+    'randomizer_getbytesfromstring_title' => 'New <code>Randomizer::getBytesFromString()</code> method',
```

Comment:
```suggestion
    'randomizer_getbytesfromstring_title' => 'New <code>Randomizer<span style="word-break: break-all;">::</span>getBytesFromString()</code> method',
```
This should fix the rendering on mobile:
![image](https://github.com/php/web-php/assets/209270/96dbc176-4b35-49cb-aad9-60797bb55aac)